app/helpers/quadicon_helper.rb

```diff
@@ -477,7 +467,7 @@ def render_host_quadicon(item, options)
       output << flobj_img_simple('100/shield.png', "g72") unless item.get_policies.empty?
     else
       output << flobj_img_simple
-      output << flobj_img_small(img_for_host_vendor(item), "e72")
+      output << flobj_img_simple(img_for_host_vendor(item), "e72", 64)
```

Comment:
Thanks, I have no real objection to removing it, especially if we can get rid of that size completely :) Just confused me here, thanks :)